### DISCUSSION TITLE
目录，摘要页的页眉问题

### DISCUSSION THREAD

**Discussion by Zhuzy0608** [note286/xduts#130] [Q&A] [answered]
目录，摘要页的页眉应该都是目录吧，不论奇数页还是偶数页

**Comment by note286** [comment]
请给出相应规定文件。

**Comment by Zhuzy0608** [comment]


**Reply by note286** [reply]
我是说论文写作规范，而不是你的论文。

**Reply by Zhuzy0608** [reply]
我是看了周围同学的，他们都是这样子，然后毕业手册里给的例子也是这样子
![image](https://github.com/note286/xduts/assets/130884066/d7d017fd-585d-4f22-bfb1-ead09bbf885e)
这个截图是2019年毕业手册里面的，不排除手册里有错，所有我来问一下

**Reply by note286** [reply, marked as answer]
你给的图对应的文件名是什么？

根据《西安电子科技大学本科生毕业设计（论文）撰写规范》：

> 页眉：宋体五号，居中排列。左面页眉为论文题目，右面页眉为章次和章标题。页眉底划线的宽度为 0.75 磅。

**Reply by note286** [reply]
最新的[毕业设计表格-全套-西安电子科技大学 教务处](https://jwc.xidian.edu.cn/info/1022/12302.htm)里没有你所述的文件。

**Reply by note286** [reply]
另外，你所提供的图中页眉一长一短，从这可以看出教务处这个文件弄得很草率，不具有什么参考价值。相比较之下，该手册所提供的文字描述更为可信一些。

![image](https://github.com/note286/xduts/assets/16207292/203f8579-d5d0-4e90-813d-4f72248d750d)

**Reply by Zhuzy0608** [reply]
明白，是我看到的文件不规范，谢谢啦
